ament_tools/build_types/cmake.py

```diff
@@ -185,7 +185,10 @@ def _common_cmake_on_test(self, context, build_type):
                     raise VerbExecutionError("Could not find 'make' executable")
                 cmd = prefix + [MAKE_EXECUTABLE, 'test']
                 if 'ARGS' not in os.environ:
-                    args = ['-V']
+                    args = [
+                        '-V',
+                        # verbose output and generate xml of test summary
+                        '-D', 'ExperimentalTest', '--no-compress-output']
```

Comment:
Following an in-person conversation with @dirk-thomas, I learned that:

* `-D ExperimentalTest` is an option to ctest to cause it to perform a kind of dashboard test that saves more information.
* `--no-compress-output` prevents that file from being archived / compressed.
* The new file can be found in `build/<pkg>/Testing/<timestamp>/Test.xml`